src/pybind/mgr/mgr_module.py

```diff
@@ -342,7 +342,59 @@ def _unit_to_str(self, unit):
             return "/s"
         elif unit == self.BYTES:
             return "B/s"  
-    
+
+    def to_pretty_iec(self, n):
+        for bits, suffix in sorted({60: 'Ei', 50: 'Pi', 40: 'Ti', 30: 'Gi',
```

Comment:
ahh, a list would be better in this case. sorry, i didn't realize that it needed to be sorted!

Reply:
yep, makes sense! updated.